pkg/machines/components/vm/consoles/consoles.jsx

```diff
@@ -41,67 +40,14 @@ const VmNotRunning = () => {
     );
 };
 
-const ConsoleSelector = ({ onChange, selected, isSerialConsole, vm }) => {
-    const entries = [];
-
-    let isDesktop = false;
-    if (vm.displays) {
-        if (vm.displays.vnc) {
-            entries.push(
-                <Select.SelectEntry data="vnc-browser" key='vnc-browser'>
-                    {_("Graphics console (VNC)")}
-                </Select.SelectEntry>
-            );
-            isDesktop = true;
-        }
-
-        if (isDesktop || vm.displays.spice) {
-            entries.push(
-                <Select.SelectEntry data="desktop" key='desktop'>
-                    {_("Graphics console in desktop viewer")}
-                </Select.SelectEntry>
-            );
-        }
-    }
-
-    if (isSerialConsole) {
-        entries.push(
-            <Select.SelectEntry data="serial-browser" key='serial-browser'>
-                {_("Serial console")}
-            </Select.SelectEntry>
-        );
-    }
-
-    return (
-        <>
-            <label htmlFor="console-type-select">{_("Console type")}</label>
-            <Select.StatelessSelect id="console-type-select"
-                                    selected={selected}
-                                    onChange={onChange}>
-                {entries}
-            </Select.StatelessSelect>
-        </>
-    );
-};
-
-const NoConsoleDefined = () => {
-    return (
-        <div>
-            {_("No console defined for this virtual machine.")}
-        </div>
-    );
-};
-
 class Consoles extends React.Component {
     constructor (props) {
         super(props);
 
         this.state = {
-            consoleType: undefined,
             consoleDetail: undefined,
         };
```

Comment:
Sorry to jump in on review so late, but I see that this state property "consoleDetail" is not used anywehere, and this component Consoles could be a stateless component